### ISSUE TITLE
Crashes on Steam Deck

### ISSUE THREAD

**Issue by lithospace** [MrCrayfish/Controllable#423]
Controllable seems to have a shader cache imcompatibility with another mod on steam deck. Makes the game crash with a rendering error on launch. I will have a list of the mods used here. Forge 40.2.1,  1.18.2.
mclo.gs/4LUQB9r

**Comment by lithospace** [comment]
mc rebuilt modpack on forge 1.18.2

**Comment by lithospace** [comment]
seems to be with iris, well in this case oculus

**Comment by MegPredator** [comment]
shader cache? you mean the regular steam shader pre-caching?

**Comment by lithospace** [comment]
Yes its unable to create a shader cache folder in a directory, causing a rendering overlay crash

**Comment by lithospace** [comment]
![20230701_231653](https://github.com/MrCrayfish/Controllable/assets/91441433/8469faee-58bc-4872-9f49-6192bfa4b9f6)

**Comment by MegPredator** [comment]
what launcher are you using?

**Comment by lithospace** [comment]
Gd launcher

**Comment by MegPredator** [comment]
try launching gd launcher in plasma desktop mode, and not from steam, from the application launcher, the gd launcher icon

**Comment by lithospace** [comment]
Same issues

**Comment by lithospace** [comment]
I've tried launching in every possible way and even from steam there should be no change

**Comment by MegPredator** [comment]
when launching outside steam, do you get the shader precache error? precaching should not even occur when launching outside steam

**Comment by lithospace** [comment]
Yes

**Comment by MegPredator** [comment]
send latest.log

**Comment by lithospace** [comment]
Also remember that this is a forge 1.18.2 instance using the mc rebuilt modpack

**Comment by lithospace** [comment]
I'm at the gym rn but when I get home I will

**Comment by MegPredator** [comment]
alright

**Comment by Paul-Ucla** [comment]
Hey im also using controllable on steam deck and i get a similar issue but im not using any other mods want to know if you have any solution ?

**Comment by lithospace** [comment]
What mods are you using and what is the instance

**Comment by Paul-Ucla** [comment]
Im only using controllable with the GDLauncher

**Comment by XadTreborp** [comment]
Also having this issue, 1.20.1, only have Controllable and it's dependency.

**Comment by MegPredator** [comment]
> Also having this issue, 1.20.1, only have Controllable and it's dependency.

Try using polyMC
